src/model.ts

```diff
@@ -168,6 +168,7 @@ export class SpraypaintBase {
   static strictAttributes: boolean = false
   static logger: ILogger = defaultLogger
   static sync: boolean = false
+  static credentials: "same-origin" | "omit" | "include" | undefined
```

Comment:
Is it worth extracting to a standalone type and remove the repetitiion that now we see at [lines 750/753](https://github.com/graphiti-api/spraypaint.js/pull/87/files#diff-6bfea2ec3c1d21c90d6e1ecd704f380a6eeb44521e1da1a87b6a48ea691b6a43R750-R753)?

Like:

```js
type Credentials = "same-origin" | "omit" | "include" | undefined
// ...
static credentials: Credentials
//...
    const options = {
      credentials: "same-origin" as Credentials,
      ...
     }
```

(Maybe it's fine as is, not the end of the world surely)